pkgs/development/tools/misc/elfutils/default.nix

```diff
@@ -74,8 +78,8 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  doCheck = false; # fails 3 out of 174 tests
-  doInstallCheck = false; # fails 70 out of 174 tests
+  doCheck = true;
+  doInstallCheck = true;
```

Comment:
Hmm, pkgsMusl.elfutils fails on both of these for me:
```
============================================================================
Testsuite summary for elfutils 0.185
============================================================================
# TOTAL: 226
# PASS:  215
# SKIP:  4
# XFAIL: 0
# FAIL:  7
# XPASS: 0
# ERROR: 0
```

and
```
done; rm -f c${pid}_.???; exit $bad
/nix/store/ys2dv0mi0nf1c0msx97qpyxn93j45wbn-bash-4.4-p23/bin/bash: line 6: 18255 Segmentation fault      (core dumped) LD_LIBRARY_PATH=/nix/store/dsi8s2y84q3zv7dzs8rdplq23rmlz1vl-elfutils-0.185/lib /nix/store/qxf42rzr7pgpbap4asn81yq0mgjfaq6p-elfutils-0.185-bin/bin/$f $opt > c${pid}_.out 2> c${pid}_.err
eu-elfclassify does not support --help
make[1]: *** [Makefile:1073: installcheck-binPROGRAMS] Error 1
```

Reply:
Yeah, I checked only native `glibc` and cross-`musl` not knowing `pkgsMusl.elfutils` allows native musl. That's nice! Checking the failures.

Reply:
A few tests fail due to too strict symbol checking in what bactrace contains. Disabled tests on musl until `elfutils` works as is upstream. Then we can fix tests there.

Reply:
Awesome, thanks for looking into it. Sounds good to me.